src/Project.ts

```diff
@@ -236,22 +260,49 @@ export default class Project {
     );
   }
 
-  public get spritesAndClones(): Sprite[] {
-    return Object.values(this.sprites)
-      .flatMap((sprite) => sprite!.andClones())
-      .sort((a, b) => a._layerOrder - b._layerOrder);
+  public addSprite(sprite: Sprite, behind?: Sprite): void {
+    if (behind) {
+      const currentIndex = this.targets.indexOf(behind);
+      this.targets.splice(currentIndex, 0, sprite);
+    } else {
+      this.targets.push(sprite);
+    }
+  }
```

Comment:
`removeSprite` below doesn't remove a sprite if it's already not in the list, but *this* function *will* double-up adding a sprite. This feels like a bad idea, even if `addSprite` is made not actually public atm lol